internal/backend/remote-state/gcs/backend_internal_test.go

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package gcs
+
+import (
+	"encoding/base64"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform/internal/backend"
+	"github.com/zclconf/go-cty/cty"
+)
+
+func TestBackendConfig_encryptionKey(t *testing.T) {
+	preCheckEnvironmentVariables(t)
+	// Cannot use t.Parallel as t.SetEnv used
+
+	// getWantValue is required because the key input is changed internally in the backend's code
```

Comment:
I know we are just copying convention used elsewhere in the codebase, but trying to parse "get want value" hurt my brain. We can leave as is, but I suspect the language we wanted to use here was `expected` instead of `want`.